config/log4j2.properties

```diff
@@ -0,0 +1,36 @@
+status = error
+name = LogstashPropertiesConfig
+
+property.filename = /tmp/logstash/logstash.log
```

Comment:
@talevy 
Why do you have replaced `log4j2.xml` by `log4j2.properties` ?
- XML is the preferred and most documented log4j2 format : look at the configuration examples here https://logging.apache.org/log4j/2.0/manual/appenders.html#RollingFileAppender
- XML format is much more concise and readable : log4j2 properties format is much more verbose with lots of redundant configuration, trying to make hierarchical configuration using flat properties.

Just an example with you rolling file appender in XML :
``` xml
<RollingFile name="RollingFile" fileName="${filename}" filePattern="/tmp/logstash/logstash-%d{MM-dd-yy-HH-mm-ss}-%i.log.gz"  >
    <Policies>
        <TimeBasedTriggeringPolicy interval="2" modulate="true" />
        <SizeBasedTriggeringPolicy size="100MB" />
    </Policies>
    <DefaultRolloverStrategy max ="5" />
    <PatternLayout pattern="[%d] %p %m (%c:%L)%n"/>
</RollingFile>
```

Isn't it more concise and readable ?